frontend/tests/controllers/ProblemCreateTest.php

```diff
@@ -829,6 +829,83 @@ public function testValidProblemInteractive() {
         );
     }
 
+    /**
+     * Test that we are able to submit a lecture problem without public tags.
+     */
+    public function testValidLecture() {
+        // Get the problem data
+        $problemData = \OmegaUp\Test\Factories\Problem::getRequest(new \OmegaUp\Test\Factories\ProblemParams([
+            'zipName' => OMEGAUP_TEST_RESOURCES_ROOT . 'readonlylecture.zip',
+            'languages' => '',
+            'selectedTags' => '',
+        ]));
```

Comment:
No sé a qué se deba que el linter no funcione correctamente en los archivos de prueba.

De cualquier forma, ¿puedes aplicar este cambio para que sea más fácil de revisar
```suggestion
        $problemData = \OmegaUp\Test\Factories\Problem::getRequest(
            new \OmegaUp\Test\Factories\ProblemParams([
                'zipName' => OMEGAUP_TEST_RESOURCES_ROOT . 'readonlylecture.zip',
                'languages' => '',
                'selectedTags' => '',
            ])
        );
```
?